t/api/10-jobgroups.t

```diff
@@ -479,4 +479,65 @@ subtest 'prevent create/update duplicate job group on same parent group' => sub(
     );
 };
 
+subtest 'update default_keep_logs_in_days and default_keep_results_in_days' => sub() {
+    my %tests_paramenters = (
+        'parent group' => {
+            endpoint => '/api/v1/parent_groups',
+            form_name => 'test_parent_group',
+            keep_logs_name => 'default_keep_logs_in_days',
+            keep_results_name => 'default_keep_results_in_days',
+        },
+        'job group' => {
+            endpoint => '/api/v1/job_groups',
+            form_name => 'test_job_group',
+            keep_logs_name => 'keep_logs_in_days',
+            keep_results_name => 'keep_results_in_days',
+        });
+
+    for my $test (keys %tests_paramenters) {
```

Comment:
I recommend to always sort hash keys, at least when running tests. It will just make the test output more predictable.
```suggestion
    for my $test (sort keys %tests_paramenters) {
```